birdhouse/components/weaver/default.env

```diff
@@ -6,6 +6,11 @@
 # single quotes are important in below list to keep variable names intact until 'pavics-compose' parses them
 EXTRA_VARS='
   $WEAVER_CONFIG
+  $WEAVER_VERSION
+  $WEAVER_MONGODB_VERSION
```

Comment:
`WEAVER_VERSION` and `WEAVER_MONGODB_VERSION` not used in `weaver.ini.template` in this PR.  Are they used outside of this PR for any sort of templating expansion?  If not then they do not have to be added to `EXTRA_VARS`, please remove.

Reply:
They are used in the extra compose of weaver component.